web/app.js

```diff
@@ -3125,6 +3121,10 @@ function onKeyDown(evt) {
   if (handled) {
     evt.preventDefault();
   }
+
+  if (this.editorUndoBar?.isOpen && evt.keyCode !== 9 && evt.keyCode !== 16) {
```

Comment:
Please add comments explaining what these keyCodes are, note the rest of this function.